crates/ty/tests/cli.rs

```diff
@@ -308,6 +308,77 @@ fn config_file_annotation_showing_where_python_version_set() -> anyhow::Result<(
     Ok(())
 }
 
+#[test]
+fn config_file_annotation_showing_where_python_version_set_syntax_error() -> anyhow::Result<()> {
```

Comment:
It would be nice if these tests could be mdtests instead. But I think our mdtest markdown parsing doesn't preserve the location

Reply:
yeah, I don't think it's worth the added tomplexity to the mdtest framework to make it possible to write these as mdtests. (Not yet, anyway -- I guess that might change if we add lots more tests like this...)

It's also actually kinda nice writing these as integration tests because the snapshots are inline 😄